File: Projeto/PythonServer/message/OperationReturn.py
import message_pb2

class OperationReturn(object):
    #--------------Constructor--------------
    def __init__(self,msg,code):
        self.__operationReturn = message_pb2.OperationReturn()
        self.__operationReturn.msg = msg
        self.__operationReturn.code = code

    #---------------------------------------

    #----------Getters and Setters----------
    def getMsg(self):
        return self.__operationReturn.msg

    def getCode(self):
        return self.__operationReturn.code

    def setMsg(self,msg):
        self.__operationReturn.msg = msg

    def setCode(self,code):
        self.__operationReturn.code = code

    #---------------------------------------

File: Projeto/Middleware/src/message/OperationReturn.java
/**
 * 
 */
package message;

import java.io.Serializable;

/**
 * @author fabio
 *
 */
public class OperationReturn implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private String msg;
	private Integer code;

	/**
	 * @param msg
	 * @param code
	 */
	public OperationReturn(String msg, Integer code) {
		this.msg = msg;
		this.code = code;
	}

	/**
	 * @return the msg
	 */
	public String getMsg() {
		return msg;
	}

	/**
	 * @param msg the msg to set
	 */
	public void setMsg(String msg) {
		this.msg = msg;
	}

	/**
	 * @return the code
	 */
	public Integer getCode() {
		return code;
	}

	/**
	 * @param code the code to set
	 */
	public void setCode(Integer code) {
		this.code = code;
	}

}
